addons/common/functions/fnc_closeDialogIfTargetMoves.sqf

```diff
@@ -13,36 +13,35 @@
 #include "script_component.hpp"
```

Comment:
I can delete it in my PR if scrapping is in order. I leave the funeral arrangements to you?